scene/2d/tile_map.compat.inc

```diff
@@ -30,10 +30,6 @@
 
 #ifndef DISABLE_DEPRECATED
 
-#include "core/object/object.h"
-
-#include "core/object/class_db.h"
-
```

Comment:
These didn't seem necessary, so I simplified. Otherwise it looked like these were ordered this way with a separation to solve a recursive inclusion issue (since object is already included in class_db), but since it compiles fine without it I guess it was a remnant from previous architectures.